slick/src/main/scala/slick/util/TreePrinter.scala

```diff
@@ -64,3 +87,18 @@ object DumpInfo {
 
   def highlight(s: String) = cGreen + s + cNormal
 }
+
+/** Create a wrapper for a `Dumpable` to omit some nodes. */
+object Ellipsis {
```

Comment:
Because that's what it creates in the tree dump (cf. https://en.wikipedia.org/wiki/Ellipsis)